<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextStyleWidgetClass</class>
 <widget class="QWidget" name="TextStyleWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextStyleWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TextStyleWidgetClass{background-color: rgb(255, 255, 255);}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>字体色</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="ColorButton" name="color_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>24</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:1px groove #dcdcdc;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:1px groove  #dcdcdc;
}

/* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 0px groove gray;
     border-top-right-radius:1px;
     border-bottom-right-radius: 1px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 12px;
 }

 QToolButton::menu-arrow {
     image: url(:/LxMemo/icons/down-arrow.png);
	width:12px;
	height:12px;
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }
QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>背景色</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ColorButton" name="bk_color_button">
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>24</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:1px groove #dcdcdc;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:1px groove  #dcdcdc;
}

/* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 0px groove gray;
     border-top-right-radius:1px;
     border-bottom-right-radius: 1px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 12px;
 }

 QToolButton::menu-arrow {
     image: url(:/LxMemo/icons/down-arrow.png);
	width:12px;
	height:12px;
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }
QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>字体</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="fontFilters">
      <set>QFontComboBox::AllFonts</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>字体大小</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>边框</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
